src/AzureStack/Admin/InfrastructureInsightsAdmin/InfrastructureInsights.Admin/Microsoft.AzureStack.Management.InfrastructureInsights.Admin.csproj

```diff
@@ -7,7 +7,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</PackageId>
         <Description>Provides developers with libraries for the health admin under AzureStack to manage health alerts.</Description>
-        <VersionPrefix>0.2.0-preview</VersionPrefix>
+        <VersionPrefix>0.3.0-preview</VersionPrefix>
         <AssemblyName>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;InfrastructureInsights;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
```

Comment:
Please update `PackageReleaseNotes`

Reply:
please mention that repair api is supported only on AzureStack 1811 +

Reply:
@dsgouda can we add a checklist item in the PR saying 'I have updated relevant package release notes' ?